delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/SupervisionRoutesSpec.scala

```diff
@@ -114,4 +128,47 @@ class SupervisionRoutesSpec extends BaseRouteSpec {
 
   }
 
+  "The projects healing endpoint" should {
+    "be forbidden without projects/write permission" in {
+      val healerWasExecuted = Ref.unsafe[IO, Boolean](false)
+      val routesWithHealer  = routesTemplate(Set.empty, projectHealer(healerWasExecuted))
```

Comment:
It would be nicer I think if the healer encapsulated the mocking code, (the ref)

Reply:
Also I wonder if it should check the specific project 🤔